### DISCUSSION TITLE
Audio dropouts with Plugins with dynamical GUI..

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#419] [Help]
I have a problem with Plugins where there is a very dynamical graphical GUI like for example DMG Limitless (the dynamic representation of the frequency spectrum/moving bands when it limits, etc.), then I very quickly get Audio dropouts, although the server has only 60% processor load. I have a MacMini 2014, so the graphic card should be not the problem here. Also when I offload the plugin from the server and load it into my DAW on my main computer, no dropouts appear, so it might be related to the AG..?

**Comment by CaBuffalo** [comment]
What's your set up?

On Tue, Mar 2, 2021 at 9:31 AM Pete3030 <notifications@github.com> wrote:

> I have a problem with Plugins where there is a very dynamical graphical
> GUI like for example DMG Limitless (the dynamic representation of the
> frequency spectrum/moving bands when it limits, etc.), then I very quickly
> get Audio dropouts, although the server has only 60% processor load. I have
> a MacMini 2014, so the graphic card should be not the problem here. Also
> when I offload the plugin from the server and load it into my DAW on my
> main computer, no dropouts appear, so it might be related to the AG..?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/419>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DIHBOYCEXQFQCSKB3TBUOIJANCNFSM4YPNHHXA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Pete3030** [reply]
Server is a macmini 2014, 2,6 ghz, 8 gb RAM and Catalina installed. Main machine is a Mac Pro 2012, with 32gb RAM, High Sierra installed, Daw is Logic.. AG runs via Ethernet switch..

**Comment by CaBuffalo** [comment]
Thanks, We have similar set up.

They offer demo. I'll try it and see what happens.

On Tue, Mar 2, 2021 at 12:01 PM Pete3030 <notifications@github.com> wrote:

> Server is a macmini 2014, 2,6 ghz, 8 gb RAM and Catalina installed. Main
> machine is a Mac Pro 2012, with 32gb RAM, High Sierra installed, Daw is
> Logic.. AG runs via Ethernet switch..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/419#discussioncomment-423010>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6D3XZO25QSTR3H4ERDTBU7Z5ANCNFSM4YPNHHXA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Pete3030** [comment]
Ok, thanks a lot! Let me know the outcome..

**Comment by Pete3030** [comment]
Did you already check if its reproducible on your side..?

**Comment by CaBuffalo** [comment]
Not yet. I'll do it tonight.

On Wed, Mar 3, 2021 at 9:43 AM Pete3030 <notifications@github.com> wrote:

> Did you already check if its reproducible on your side..?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/419#discussioncomment-426569>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DLOD2AJ3PCB56FQ3DTBZYNDANCNFSM4YPNHHXA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Pete3030** [comment]
ok, thanks..

**Comment by CaBuffalo** [comment]
I just tested it. It's been more than half an hour. There is no drop out
that happens to me.

On Wed, Mar 3, 2021 at 11:09 AM Pete3030 <notifications@github.com> wrote:

> ok, thanks..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/419#discussioncomment-426889>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CIZSP7GKBE2RMZC63TB2CQBANCNFSM4YPNHHXA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Pete3030** [reply]
I made a short video demonstrating the issue. I am on Logic 10.4.8 btw. The CPU core is spiking as soon as I open the Plugin Gui, very strange..

https://user-images.githubusercontent.com/65017275/110203678-cebe9c00-7e6f-11eb-8180-a2d7fb4b538f.MOV

**Reply by Pete3030** [reply]
What is interesting, when I open the plugin within a second AG second server which is the one on my DAW computer, I do not get any spikes/dropouts. At least this is a workaround for me, because this solution still eats a lot less cpu than opening the plugin in the DAW itself..;)
